kubetest/anywhere.go

```diff
@@ -79,6 +79,8 @@ type kubernetesAnywhere struct {
 	Cluster           string
 }
 
+var _ deployer = kubernetesAnywhere{}
```

Comment:
Yes, exactly. I should've added them initially, and it's not a huge deal here because they're all part of the same package, but this provides a check in the file that's being compiled that `kubernetesAnywhere{}` satisfies the `deployer` interface. Otherwise you end up with an error in `main.go` - which is still readable, but you might as well assert it here. (This will also help if we split these up into packages in separate repos later.)